### PR TITLE
[CoSim] sdof mpi

### PR DESCRIPTION
**Pull request by AndreasWinterstein**
Adds fixes for mpi to sdof solver for data transfer operators and output.

### CODE REVIEW

File: applications/CoSimulationApplication/python_scripts/data_transfer_operators/copy_single_to_distributed.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
 
+import KratosMultiphysics as KM
```
Comment (philbucher) [inline applications/CoSimulationApplication/python_scripts/data_transfer_operators/copy_single_to_distributed.py]:
```suggestion
```

**Comment by adityaghantasala** [comment]
Looks good. 
A minor thing that is not related to the PR,  but should be considered is , the name of the class `CopySingleToDistributed`.  In a first glance, `Distributed` looks like it has to do something about MPI distribution. 
So a better name could be `CopyScalarToField` or something similar is more descriptive IMO.

**Comment by philbucher** [comment]
> A minor thing that is not related to the PR, but should be considered is , the name of the class `CopySingleToDistributed`. In a first glance, `Distributed` looks like it has to do something about MPI distribution.

I thought the same when I looked at it.
I suggest to change it when we have the sub-comms, then we anyway will need to do some refactoring

**Comment by adityaghantasala** [comment]
@philbucher  we can and should do it in a different PR. 
Can you add this to the Co-sim TODO list. 
Also Irrespective if we have split coms or not the name I think is somehow misleading.

**Review by philbucher — APPROVED** [review]

**Comment by philbucher** [comment]
@AndreasWinterstein can we delete this branch?

**Pull request merged**